src/zzz_od/application/inventory_scan/utils/agent_icon_matcher.py

```
@@ -0,0 +1,282 @@
+            else:
+                log.warning(f"未找到对应的代理人: {icon_name}")
+
+            self._cache_unique_crop_debug(
+                small_icon=small_icon,
+                best_icon_name=best_icon_name,
+                best_score=best_score,
+                agent_key=agent_key or "",
+                top_scores=scores[:5]
+            )
+
+            return agent_key or ""
```

Ruff (0.15.4)

[warning] Consider moving this statement to an `else` block

(TRY300)